core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.html

```diff
@@ -403,38 +534,176 @@
       from { transform: translateY(-100%); }
       to { transform: translateY(0); }
   }
-  @media only screen and (max-width: 600px) {
-    .oppia-opportunities-tabs-list {
+  @media only screen and (max-width: 700px) {
```

Comment:
Is there any way to create constants for these breakpoints and import them?

Reply:
I'm not sure if there is a way to import variables into css from a typescript file. Perhaps we could do something similar to [this](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)?

Reply:
Right, I was thinking we could use native CSS variables.